src/patterns/components/alerts/collection.yaml

```diff
@@ -9,7 +9,61 @@ restrictions:
   - Should fill the width of the viewport.
   - The content should be kept short and concise.
   - role="alert" is required so that assistive technology can inform users their attention is needed.
-  - The data-id property is provided as a hook for automated tools. If you have multiple 
-    instances of the same variant of a component on the same page, make sure each instance 
+  - The data-id property is provided as a hook for automated tools. If you have multiple
+    instances of the same variant of a component on the same page, make sure each instance
     has a unique data-id property ("alert-info-1", "alert-info-2", "alert-info-1", etc).
 sparkPackageCore: true
+variableTable:
+  $alert-border:
+    default: 1px solid $gray
+    description: Sets the border surrounding the alert.
+  $alert-bar-width:
+    default: -10px
+    description: Sets the width of the inset box-shadow X offset value to create the color bar on the right side of the alert.
```

Comment:
Sets the width of the color bar on the right side of the alert.

or

Sets the width of the box-shadow used to create the color bar on the right side of the alert.